taichi/analysis/gather_uniquely_accessed_pointers.cpp

```diff
@@ -120,6 +166,9 @@ class UniquelyAccessedSNodeSearcher : public BasicStmtVisitor {
   std::unordered_map<const SNode *, GlobalPtrStmt *> accessed_pointer_;
   std::unordered_map<const SNode *, GlobalPtrStmt *> rel_access_pointer_;
 
+  // Search any_arrs that are uniquely accessed. Maps: ArgID -> ExternalPtrStmt
```

Comment:
What is `any_arrs`? Is it just "any external arrays"?

Reply:
ext_arr & ndarrays -> `ti.any_arr`